payonidhi/if_else/lower_upper_case.cpp

```diff
@@ -0,0 +1,18 @@
+#include <iostream>
+using namespace std;
+int main() {
+  char ch;
+  cout<<"Enter any alphabet"<<endl;
+  cin>>ch;
+  int x;
+  x = ch;
+  if((x>=97) && (x<=122)){
```

Comment:
Most of the time we forget what this code will do because of readability factor. You can do if(ch >= 'A' && ch <= 'Z'), it is more readable and don't need to remember ascii table as well.